azkaban-db/src/main/sql/upgrade.3.42.0.to.3.43.0.sql

```diff
@@ -0,0 +1,13 @@
+-- DB Migration from release 3.42.0 to 3.43.0
+-- PR #1657 changes project cache to case insensitive.
+--
+-- 1. When creating new database,
+-- use below query to explicitly set the COLLATION to case insensitive for the entire database:
+--
+ALTER DATABASE <database_name> CHARACTER SET utf8 COLLATE utf8_general_ci;
```

Comment:
https://docs.microsoft.com/en-us/sql/relational-databases/collations/set-or-change-the-database-collation:
`You can change the collation of any new objects that are created in a user database by using the COLLATE clause of the ALTER DATABASE statement. This statement does not change the collation of the columns in any existing user-defined tables. These can be changed by using the COLLATE clause of ALTER TABLE.`

Reply:
So both mysql and h2 are verified that this query can work?

Reply:
No, this query only works for Mysql. H2 and Mysql have different syntax in setting collation (case sensitivity).

Reply:
Please leave a comment that this is DB specific.

Reply:
Sure, added the comment in the new commit.